src/coreclr/jit/hwintrinsicarm64.cpp

```diff
@@ -547,7 +547,12 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
         case NI_Vector128_Ceiling:
         {
             assert(sig->numArgs == 1);
-            assert(varTypeIsFloating(simdBaseType));
+
+            if (!varTypeIsFloating(simdBaseType))
+            {
+                retNode = impSIMDPopStack();
+                break;
+            }
```

Comment:
What's the reason for these JIT changes?  Was something breaking without it? Are we missing any tests, or it's newly exposed by the interfaces and is thus implicitly tested?